### DISCUSSION TITLE
brew install mariadb

### DISCUSSION THREAD

**Discussion by Lomlomm** [Homebrew/discussions#2154] [Everyday usage]
Im not able to install mariadb/mysql with homebrew, does anyone knows if there is any solution yet? 
I already tried with the following commands:

brew install mariadb 
arch -x86_64 brew install mariadb
brew install mysql
brew install mysql@5.7

This is the output I get:

fatal: Could not resolve HEAD to a revision
Warning: No available formula or cask with the name "mariadb".
==> Searching for similarly named formulae...
Error: No similarly named formulae found.

**Comment by SMillerDev** [comment]
What does `brew doctor` say?
